pkg/cli/image/mirror/mirror.go

```diff
@@ -764,9 +770,6 @@ func copyManifest(
 	for _, desc := range srcManifest.References() {
 		plan.parent.parent.AssociateBlob(desc.Digest, plan.parent.name)
 	}
-	if srcDigest != toDigest {
-		fmt.Fprintf(errOut, "warning: Digests are not preserved with schema version 1 images. Support for schema version 1 images will be removed in a future release.\n")
-	}
```

Comment:
Digest passed != digest returned with 'oc image mirror multi-arch-image'  but it's not because it was converted from schema1 to 2, this is wrong. Same as above.